config/application.rb

```diff
@@ -100,15 +100,13 @@ class Application < Rails::Application
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
 
-    # Custom directories with classes and modules you want to be autoloadable.
-    # config.autoload_paths += %W(#{config.root}/extras)
-    config.autoload_paths += Dir["#{config.root}/lib"]
-    config.autoload_paths += Dir["#{config.root}/app/controllers/concerns"]
-    config.autoload_paths += Dir[Rails.root.join('app', 'models', 'power_manager')]
-    config.autoload_paths += Dir["#{config.root}/app/models/concerns"]
-    config.autoload_paths += Dir["#{config.root}/app/services"]
-    config.autoload_paths += Dir["#{config.root}/app/mailers"]
-
+    # Autoloading
+    config.autoload_paths += %W(#{config.root}/lib)
+    config.autoload_paths += %W(#{config.root}/app/controllers/concerns)
+    config.autoload_paths += %W(#{config.root}/app/models/power_manager)
+    config.autoload_paths += %W(#{config.root}/app/models/concerns)
+    config.autoload_paths += %W(#{config.root}/app/services)
+    config.autoload_paths += %W(#{config.root}/app/mailers)
```

Comment:
No change here I am just rewriting it so all lines in this block use the same syntax.

From my testing, it looks like autoloading, at least in Rails version which is in use, is recursive. I created `lib/test/test.rb` and it autoloaded the class from there.

Reply:
it definitelly looks much better, I thing that this would look even better, but that's only my preference.
```
config.autoload_paths << config.root.join('app', 'mailers')
```